content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -0,0 +1,116 @@
+---
+html: get-started-evm-sidechain.html
+parent: evm-sidechains.html
+blurb: Get started with the EVM compatible sidechain for the XRP Ledger.
+labels:
+  - Development, Interoperability
+status: not_enabled
+---
+# Get Started with the EVM Sidechain
+
+This getting started tutorial walks you through the steps to set up your account and transfer funds using the EVM sidechain bridge. 
+
+## 1. Create an Account Using an EVM Compatible Wallet
+
+In order to interact with the network, you need to create an account in the EVM sidechain. To create and manage this account you can use any EVM compatible wallet such as MetaMask.
+
+For instructions on how to install and create an account using MetaMask, and then send and receive tokens, see [Connect MetaMask to XRPL Sidechain](connect-metamask-to-xrpl-evm-sidechain.html).
+
+## 2. Move XRP Ledger Devnet Tokens to the EVM Sidechain
+
+Before you can start interacting with the EVM blockchain, you need to transfer some tokens from the XRP Ledger Devnet to the EVM sidechain. 
+
+To obtain tokens in the XRP Ledger Devnet, go to the [XRP Faucets](xrp-testnet-faucet.html) page and click *Generate Devnet credentials* to generate a new Devnet account with some test XRP in it as shown in the following image.
```

Comment:
I don't think you need a screenshot for this. It's something we would have to maintain. Regardless, I wouldn't say "as shown in the following image."